src/main/webapp/app/modules/conflicts/shared/components/input-field.tsx

```diff
@@ -225,6 +230,22 @@ export class InputField extends React.Component<IInputFieldProp, IInputFieldStat
       );
     }
 
+    let urlButton = null;
+    if (fieldName === 'url' && defaultValue) {
+      urlButton = (
+        <div className="url-icon" onClick={this.handleUrl(this.props.activity.organization.url)}>
```

Comment:
Cant you just put FontAwesomeIcon inside anchor tag? Whats the point of having clickable div (which handles url) and anchor tag inside it?